yarn.lock

```diff
@@ -4358,6 +4359,13 @@ magic-string@^0.25.2:
   dependencies:
     sourcemap-codec "^1.4.4"
 
+make-dir@^1.0.0:
```

Comment:
Not quite sure why this was added, there is no dependency in `linguist-languages`.

Reply:
:confused: , i will look on this